SpatialGDK/Source/SpatialView/SpatialView.Build.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+using UnrealBuildTool;
+
+public class SpatialView : ModuleRules
+{
+	public SpatialView(ReadOnlyTargetRules Target) : base(Target)
+	{
+		PCHUsage = PCHUsageMode.UseExplicitOrSharedPCHs;
+		bFasterWithoutUnity = true;
+
```

Comment:
Do we also need to link the worker sdk here?

Reply:
We do, my intention was to start minimal and add what's needed as we go.